packages/server/api/src/app/flows/folder/folder-flows.ts

```diff
@@ -0,0 +1,59 @@
+import { ApplicationError, ContentType, ErrorCode } from '@openops/shared';
```

Comment:
okay im pretty happy with the change now, it minimizes duplication and we can reuse getFilterCondition also for uncategorized (the only thing im not happy with the is name getFilterCondition)